src/components/UI/SubmitBTN.tsx

```diff
@@ -0,0 +1,24 @@
+import { twMerge } from 'tailwind-merge';
+import Spinner from './Spinner';
+
+interface submitBTN {
+  text: string;
+  styles?: string;
+  handlesubmit: () => void;
+}
+
+const SubmitBTN: React.FC<submitBTN> = ({ text, handlesubmit, styles }) => {
```

Comment:
can we set styles default value as ```""``` so we dont need to add a extra shortcouit selector?